tests/components/smhi/conftest.py

```diff
@@ -1,25 +1,131 @@
 """Provide common smhi fixtures."""
 
+from __future__ import annotations
+
+from collections.abc import AsyncGenerator, Generator
+import json
+from typing import Any
+from unittest.mock import AsyncMock, MagicMock, patch
+
+from pysmhi.smhi_forecast import SMHIForecast, SMHIPointForecast
 import pytest
 
+from homeassistant.components.smhi import PLATFORMS
 from homeassistant.components.smhi.const import DOMAIN
+from homeassistant.const import CONF_LATITUDE, CONF_LOCATION, CONF_LONGITUDE, Platform
+from homeassistant.core import HomeAssistant
+
+from . import TEST_CONFIG
+
+from tests.common import MockConfigEntry, load_fixture
+from tests.test_util.aiohttp import AiohttpClientMocker
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[AsyncMock]:
+    """Override async_setup_entry."""
+    with patch(
+        "homeassistant.components.smhi.async_setup_entry", return_value=True
+    ) as mock_setup_entry:
+        yield mock_setup_entry
+
+
+@pytest.fixture(name="load_platforms")
+async def patch_platform_constant() -> list[Platform]:
+    """Return list of platforms to load."""
+    return PLATFORMS
+
+
+@pytest.fixture
+async def load_int(
+    hass: HomeAssistant,
+    mock_client: SMHIPointForecast,
+    load_platforms: list[Platform],
+) -> MockConfigEntry:
+    """Set up the SMHI integration."""
+    hass.config.latitude = "59.32624"
+    hass.config.longitude = "17.84197"
```

Comment:
Impressive that they have a busstop in the middle of nowhere
![image](https://github.com/user-attachments/assets/030145d8-cae8-42ed-bdbc-e91422f52014)

Reply:
Haha. No clue what those coords points to 🤣